automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeCreationDiskSpec.scala

```diff
@@ -219,19 +220,29 @@ class RuntimeCreationDiskSpec
           }
         })
         _ <- deleteRuntimeWithWait(googleProject, runtimeName, false)
+        _ <- LeonardoApiClient.patchDisk(googleProject, diskName, UpdateDiskRequest(Map.empty, newDiskSize))
+        _ <- IO.sleep(5 seconds)
 
         // Creating new runtime with existing disk should have test.txt file and user installed package
         runtimeWithData <- createRuntimeWithWait(googleProject, runtimeWithDataName, createRuntime2Request)
         clusterCopyWithData = ClusterCopy.fromGetRuntimeResponseCopy(runtimeWithData)
         _ <- IO(withWebDriver { implicit driver =>
-          withNewNotebook(clusterCopyWithData, Python3) { notebookPage =>
-            val persistedData =
-              """! cat /home/jupyter/test.txt""".stripMargin
-            notebookPage.executeCell(persistedData).get should include("this should save")
-            val persistedPackage = "! pip show beautifulSoup4"
-            notebookPage.executeCell(persistedPackage).get should include(
-              "/home/jupyter/.local/lib/python3.7/site-packages"
-            )
+          withNewNotebook(clusterCopyWithData, Python3) {
+            notebookPage =>
+              val persistedData =
+                """! cat /home/jupyter/test.txt""".stripMargin
+              notebookPage.executeCell(persistedData).get should include("this should save")
+              val persistedPackage = "! pip show beautifulSoup4"
+              notebookPage.executeCell(persistedPackage).get should include(
+                "/home/jupyter/.local/lib/python3.7/site-packages"
+              )
+
+              val res = notebookPage
+                .executeCell(
+                  "! df -H |grep sdb"
```

Comment:
nit: I landed on this command to get a more precise value (size only, home dir only):

```
df -h --output=size $HOME
```